utils/users.py

```diff
@@ -49,4 +50,60 @@ def update_trendy_status(user):
             user.addType(user.TRENDY)
         else:
             user.removeType(user.TRENDY)
-        session.commit()
\ No newline at end of file
+        session.commit()
+
+def get_user_warnings_count(user):    #returns warnings for any given user
+    with Session(common.database) as session:
+            warnings_count= user.warnings
+            return warnings_count
+    
+def handle_three_warnings(user):
```

Comment:
Make a separate endpoint for handling user paying to get rid of the warnings instead of whatever this is.